frontend/src/static/css/style.scss

```diff
@@ -252,22 +258,41 @@ header{
         }
 
         .untouched{
-            $color:mui-color("black");
-            border-style:solid;
-            border-color:$color;
-            border-width:1px 0;
             margin-top:1rem;
 
             .closer{
                 cursor:pointer;
                 padding:0.5rem 1rem;
 
-                span{
-                    color:$color;
-                    font-weight:bold;
+                .fa-ellipsis-h{
+                    background-color: #bfbdbd;
+                    color: white;
+                    width: 2.5em;
+                    border-radius: 5px;
```

Comment:
eh, can standardize? cos all the previous we using rem, one rem is roughly 16px on screen same for below

Reply:
**minor nit**: eh, can choose to do as well, but its kinda easier to read if your properties are grouped

- width+height+position+display
- background stuff
- flex properties
- padding+margin
- font stuff+color
- other random stuff 

same below